apps/game/lib/game/cup.ex

```diff
@@ -0,0 +1,12 @@
+defmodule Cup do
+  @moduledoc """
+  generat a hand of n dices in a cup
+  """
+
+  @spec new(integer) :: [integer]
```

Comment:
`integer` means a single integer
`[integer]` means a list of integers
That's my understanding. Please correct me if I'm wrong. Thanks